doc/design/OrionHDFSSink.md

```diff
@@ -202,6 +202,27 @@ Provisions a Hive table with data stores in column-like mode within the given HD
 
 [Top](#top)
 
+###<a name="section4.3"></a>Authentication and authorization
+The sink for HDFS accepts  tokens as authentication and authorization mechanism. This can be configured through , as seen before.
```

Comment:
Something seems to be missing just after "This can be configured through". BTW, there is an extra whitespace between "accepts" and "tokens".